hi-ml-histopathology/.vscode/settings.json

```diff
@@ -0,0 +1,70 @@
+{
+    // The settings below are shared across all projects.
+    // When making changes here, please ensure they are kept in sync
+    // across all projects and the workspace configuration.
+    "autoDocstring.docstringFormat": "sphinx-notypes",
+    "fileHeaderComment.parameter": {
+      "*": {
+        "line": "-------------------------------------------------------------------------------------------",
+        "commentprefix": "#",
+        "company": "Microsoft Corporation"
+      }
+    },
+    "fileHeaderComment.template": {
+      "*": [
+        "${commentprefix}  ${line}",
+        "${commentprefix}  Copyright (c) ${company}. All rights reserved.",
+        "${commentprefix}  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.",
+        "${commentprefix}  ${line}"
+      ]
+    },
+    "files.trimTrailingWhitespace": true,
+    "files.insertFinalNewline": true,
+    "files.trimFinalNewlines": true,
+    "python.linting.pylintEnabled": true,
+    "python.linting.pylintArgs": [
+      "--max-line-length=120"
+    ],
+    "python.linting.flake8Enabled": true,
+    "python.linting.mypyEnabled": true,
+    "python.linting.pycodestyleEnabled": false,
+    "python.linting.pycodestyleArgs": [
+      "--max-line-length=120",
+      "--show-source",
+      "--show-pep8"
+    ],
+    "python.formatting.provider": "black",
+    "python.formatting.blackArgs": [
+      "--line-length=120"
+    ],
+    "python.testing.unittestEnabled": false,
+    "python.testing.pytestEnabled": true,
+    "rewrap.autoWrap.enabled": true,
+    "rewrap.wrappingColumn": 120,
+    "[python]": {
+      "editor.rulers": [
+        120
+      ]
+    },
+    // The settings below are specific to this project,
+    // and may override settings in the workspace configuration.
+    "python.sortImports.args": [
+        "-l=120",
+        "--src=${workspaceFolder}/../hi-ml/src",
+        "--src=${workspaceFolder}/../hi-ml-azure/src",
+        "--src=${workspaceFolder}/src"
+    ],
+    "python.testing.pytestArgs": [
+        "${workspaceFolder}/testhisto",
+        "${workspaceFolder}/testSSL"
+    ],
+    "python.analysis.extraPaths": [
+        "./hi-ml-azure/src",
```

Comment:
Are these unaffected by the workspaceFolder?

Reply:
no, you spotted a problem, fixed!